CMakeLists.txt

```diff
@@ -186,11 +186,11 @@ add_subdirectory(tools)
 add_subdirectory(test)
 add_subdirectory(integration_test)
 
-#option(CIRCT_INCLUDE_DOCS "Generate build targets for the CIRCT docs."
-#  ${LLVM_INCLUDE_DOCS} ${MLIR_INCLUDE_DOCS})
-#if (CIRCT_INCLUDE_DOCS)
-#  add_subdirectory(docs)
-#endif()
+option(CIRCT_INCLUDE_DOCS "Generate build targets for the CIRCT docs."
+  ${LLVM_INCLUDE_DOCS} ${MLIR_INCLUDE_DOCS})
```

Comment:
I'm not sure what the expected behavior here is.  What happens if LLVM_INCLUDE_DOCS=on and MLIR_INCLUDE_DOCS=on?

Reply:
I just uncomment these lines, Currently, CIRCT is out of llvm-project repository and it does not depend on the LLVM_INCLUDE_DOCS and MLIR_INCLUDE_DOCS to be set ON. So we can remove these variables for now.